client/src/components/imgdetail/Imgdetail.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from "./ImgDetail.module.css";
+
+function ImgDetails() {
+  return (
+    <div className={styles.container}>
+      <img
+        className={styles.img}
+        src="https://media.istockphoto.com/id/481229372/fr/photo/galaxie-spirale-illustration-de-la-voie-lact%C3%A9e.jpg?s=612x612&w=0&k=20&c=b2PsgdKm7IwiJfrGbhq2SgNbPXrgczuE3QPQ712g5Z4="
+        alt="test"
+      />
+      <h1 className={styles.title}> Spiral Galaxy </h1>
+      <p className={styles.description}>
+        {" "}
```

Comment:
Remove this

Reply:
je me permet de répondre à la place de Chaymae de base elle n'avait mit aucun '{" "}' mais à l'analyse du code au commit prettier lui demande de rajouter ça pour validé le commit